osf/models/preprint_provider.py

```diff
@@ -87,11 +85,11 @@ def rules_to_subjects(rules):
     q = []
     for rule in rules:
         if rule[1]:
-            q.append(Q('parent', 'eq', Subject.load(rule[0][-1])))
+            q.append(models.Q(parent=Subject.load(rule[0][-1])))
             if len(rule[0]) == 1:
-                potential_parents = Subject.find(Q('parent', 'eq', Subject.load(rule[0][-1])))
+                potential_parents = Subject.objects.filter(parent=Subject.load(rule[0][-1]))
                 for parent in potential_parents:
-                    q.append(Q('parent', 'eq', parent))
+                    q.append(models.Q(parent=parent))
         for sub in rule[0]:
-            q.append(Q('_id', 'eq', sub))
-    return Subject.find(reduce(lambda x, y: x | y, q)) if len(q) > 1 else (Subject.find(q[0]) if len(q) else Subject.find())
+            q.append(models.Q(_id=sub))
+    return Subject.objects.filter(reduce(lambda x, y: x | y, q)) if len(q) > 1 else (Subject.objects.filter(q[0]) if len(q) else Subject.objects.all())
```

Comment:
We talked about this a bit in person, but adding it here for completeness!

This method used in the admin app tests and subject methods used right now for creating/editing Preprint Providers that don't have custom taxomomies. Since there are current tests that expect the objects instead of the ids, and this method and those tests will be removed as soon as everything has custom taxonomies and subjects to rules goes away entirely, leaving this for now.